<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameRoomWidget</class>
 <widget class="QWidget" name="GameRoomWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>140</y>
     <width>258</width>
     <height>224</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextBrowser" name="chat_textBrowser"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="msg_lineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="send_pushButton">
        <property name="text">
         <string>发送消息</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
